backend/FwLite/FwDataMiniLcmBridge/LcmUtils/ProjectLoader.cs

```diff
@@ -16,8 +17,12 @@ public interface IProjectLoader
 
 public class ProjectLoader : IProjectLoader
 {
-    public const string ProjectFolder = @"C:\ProgramData\SIL\FieldWorks\Projects";
-    private static string TemplatesFolder { get; } = @"C:\ProgramData\SIL\FieldWorks\Templates";
+    public static string DataFolder =
+        RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ?
+            @"C:\ProgramData\SIL\FieldWorks" :
+            Path.Join(Environment.GetEnvironmentVariable("XDG_DATA_HOME") ?? Path.Join(Environment.GetEnvironmentVariable("HOME"), ".local", "share"), "fieldworks");
+    public static string ProjectFolder = Path.Join(DataFolder, "Projects");
```

Comment:
this property and `DataFolder` should both be `readonly`

Reply:
Done in commit 99e250a2.